enclave_wrangler/README.md

```diff
@@ -1,51 +1,52 @@
-## Enclave wrangler
+## **Enclave Wrangler**
```

Comment:
Bolding in using `**` inside of a header (e.g. an H2 in this case: `##`) will not have an affect on how the markdown appears when rendered, at least not in GitHub. So we can leave the `**` out.

I can make this change after merging!